bin/auth_ejabberd.php

```diff
@@ -48,8 +48,13 @@
 
 require dirname(__DIR__) . '/vendor/autoload.php';
 
+// BC: Add console command as second argument
+$argv = $_SERVER['argv'] ?? [];
+array_splice($argv, 1, 0, "auth_ejabberd");
+$_SERVER['argv'] = $argv;
+
 $container = \Friendica\Core\DiceContainer::fromBasePath(dirname(__DIR__));
 
 $app = \Friendica\App::fromContainer($container);
```

Comment:
I cannot add a deprecation warning.
Jabberd is using this file DIRECTLY and passing input per STDIN.
I don't know if jabberd can be configured to use a binary including a sub-command ...

Reply:
That's ok for now I think. Maybe you can add the comment section from the `src/Security/ExAuth.php` file here?

Reply:
It's already there, or what do you mean? :)

Reply:
You are right. I only checked the diffs, but the comment section was already there. Sorry for bothering.